src/main/java/artifacts/mixin/mixins/statuseffect/LivingEntityMixin.java

```diff
@@ -30,13 +31,15 @@ public LivingEntityMixin(EntityType<?> type, World world) {
 	private void applyPermanentEffects(CallbackInfo info) {
 		if (!this.world.isClient && this.age % 15 == 0) {
 
-			TrinketsHelper.getAllEquipped((LivingEntity) (Object) this).forEach(stack -> {
+			TrinketsApi.getTrinketComponent((LivingEntity) (Object) this).ifPresent(comp -> comp.getAllEquipped().forEach(pair -> {
```

Comment:
This is not the same: our TrinketsHelper only returns stacks for Artifacts trinkets that are enabled, using the trinkets API you're getting all trinkets including from other mods.